sleap/gui/app.py

```diff
@@ -1120,6 +1148,35 @@ def _update_gui_state(self):
 
         # Update menus
 
+        def _update_sessions_menu(self, frame_idx):
+            """Update the instance groups menu based on the selected frame index in the session."""
+            session = self.state.get("session")
+            if not session or frame_idx >= len(session.frame_groups):
+                return 
+
+            frame_group = session.frame_groups[frame_idx]
+
+        frame_idx = 0
```

Comment:
We can also get the `frame_idx` from the `GuiState`:
```suggestion
            frame_idx = self.state["frame_idx"]
```
Yo... this needs to be in the `_update_sessions_menu` function (add a tab)...